checks/grouping.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
+from astropy.table import Table
+from astropy.table import Table
+from astropy.coordinates import Angle, SkyCoord
+from gammapy.data import DataStore
+from gammapy.data import ObservationGroupAxis
+from gammapy.data import ObservationGroups
+from gammapy.datasets import gammapy_extra
+import pylab as pt
+pt.ion()
+
+dir=str(gammapy_extra.dir)+'/datasets/hess-crab4'
+data_store = DataStore.from_dir(dir)
+Observation_Table=data_store.obs_table
+alt = Angle([0, 30, 60, 90], 'degree')
+az = Angle([-90, 90, 270], 'degree')
+ntels = np.array([3, 4])
+list_obs_group_axis = [ObservationGroupAxis('ALT', alt, 'bin_edges'),
+                       ObservationGroupAxis('AZ', az, 'bin_edges'),
+                       ObservationGroupAxis('N_TELS', ntels, 'bin_values')]
+obs_groups = ObservationGroups(list_obs_group_axis)
+obs_table_grouped = obs_groups.group_observation_table(Observation_Table)
```

Comment:
The first solution consist of changing the file make_tables.py in hess-crab4 in order to add in the observation table the ALT and the muonefficiency column right?
Le 21 janv. 2016 à 08:38, Christoph Deil <notifications@github.com> a écrit :

> In checks/grouping.py:
> 
> > +from gammapy.data import ObservationGroups
> > +from gammapy.datasets import gammapy_extra
> > +import pylab as pt
> > +pt.ion()
> > +
> > +dir=str(gammapy_extra.dir)+'/datasets/hess-crab4'
> > +data_store = DataStore.from_dir(dir)
> > +Observation_Table=data_store.obs_table
> > +alt = Angle([0, 30, 60, 90], 'degree')
> > +az = Angle([-90, 90, 270], 'degree')
> > +ntels = np.array([3, 4])
> > +list_obs_group_axis = [ObservationGroupAxis('ALT', alt, 'bin_edges'),
> > +                       ObservationGroupAxis('AZ', az, 'bin_edges'),
> > +                       ObservationGroupAxis('N_TELS', ntels, 'bin_values')]
> > +obs_groups = ObservationGroups(list_obs_group_axis)
> > +obs_table_grouped = obs_groups.group_observation_table(Observation_Table)
> Whet do you think it’s the best?
> 
> Whatever solution is fastest to implement for you is OK with me.
> The important thing is to have an example file in gammapy-extra that can be used for tests and docs.
> 
> —
> Reply to this email directly or view it on GitHub.
>